cmd/lakectl/cmd/branch_tools.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/treeverse/lakefs/pkg/api"
+	"github.com/treeverse/lakefs/pkg/metastore"
+	"github.com/treeverse/lakefs/pkg/uri"
+)
+
+var branchCreatedTemplate = `created branch "{{.Branch.Ref}}" {{.Resp}}
+`
+
+// MissingDBError means that the requested DB is missing/not found
+type MissingDBError struct {
+	dbName string
+	err    error
+}
+
+func (mdbe MissingDBError) Error() string {
```

Comment:
That's a lot of work to create custom errors - and I'm not sure why they are needed.

Can something like the following work?
```
fmt.Errorf("trying to reach database: %w", originalErr)
```

Reply:
This was done to test the code not by the string but by the type of the error...
Go doesn't consider custom errors to be a good practice? (truly don't know 😅 )